RenameVariables: Exclude variables from renaming (#543)
MarketSquare/robotframework-tidy: docs/source/transformers/RenameVariables.rst

Variable case in ``*** Keywords ***``, ``*** Tasks ***`` and ``*** Test Cases ***`` sections depends on the
variable scope. Local variables are lowercase and global variables are uppercase. Any unknown variable (not defined

> in current keyword or test case) is considered as global. You can configure what should happen with unknown variables using

Reviewer: ```suggestion
in the current keyword or test case) is considered as global. You can configure what should happen with unknown variables using
```

                ${local}    Set Variable    value  # since we set it, ${local} is also known
                Keyword Call    ${arg}    ${local}    ${global}  # ${global} is unknown

Ignore variable case
--------------------

> All variables case are converted according to configured conventions. It is possible to pass names of the variables
> that should be ignored. By default following variables case is ignored and not transformed:

Reviewer: ```suggestion
Case of all variables is converted according to the configured conventions. It is possible to pass the names of the
variables that should be ignored. By default, the following variables case is ignored and not transformed:
```

> - ${\n}

Reviewer: Is it a complete list?

Author: Yes. It could be longer if I decide to ignore True etc. But if not I will reword this part of the docs

> Configure ``ignore_case`` to ignore additional list. This parameter accept comma separated list of variable names,
> case sensitive::

Reviewer: ```suggestion
Configure ``ignore_case`` to ignore an additional list. This parameter accepts comma-separated list of variable names
(case-sensitive)::
```
